apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -340,6 +347,11 @@ class PageTemplateModal extends Component {
 			return null;
 		}
 
+		// Skip rendering current preview if there is no page content.
+		if ( isCurrentPreview && ! blocksByTemplateSlug.current?.length ) {
+			return null;
+		}
```

Comment:
Similarly, this is what I had added as the condition to hide the 'Current' section on a new page.  I was assuming that a new page would always have no content, or that if it did have content we wouldn't want to hide 'Current' ? 🤔   Similarly if it is not a new page and has no content, we don't need the 'current' section.

Reply:
I think these are valid assumptions. We probably don't need "Current" for any empty page. New or not.